### PR TITLE
Update Jenkinsfile to last version of pipeline

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -52,9 +52,14 @@ def presubmit(gitUtils, bazel, utils) {
...
     stage('Integration Tests') {
       timeout(15) {
-        sh('bin/e2e.sh -tag alpha' + gitUtils.GIT_SHA + ' -v 2')
+        sh("bin/e2e.sh -tag alpha${gitUtils.GIT_SHA} -v 2")
       }
     }
+    stage('Docker Test Push'){
+      def images = 'init,runtime'
+      def tags = gitUtils.GIT_SHA
+      utils.publishDockerImagesToContainerRegistry(images, tags)
```
Comment (kyessenov) [inline Jenkinsfile]: I think e2e already publishes images, take a look at the script.

Comment (kyessenov) [inline Jenkinsfile]: (the assumption in e2e is that cluster is not local, so we move images to a central repo first)

Comment (sebastienvas) [inline Jenkinsfile]: You are right. Can I rename alphaSHA to testing-SHA?

**Comment by sebastienvas** [comment]
Jenkins please build istio/manager-pr-master

**Comment by sebastienvas** [comment]
will create another PR for istioctl release.

**Pull request closed without merging**